ci/scripts/get-bitstream-strategy.sh

```diff
@@ -33,11 +33,15 @@ else
 fi
 
 # Retrieve the most recent bitstream at or before HEAD.
-BITSTREAM="--refresh HEAD" ./bazelisk.sh build ${bitstream_target}
+BITSTREAM="--refresh HEAD" ./bazelisk.sh build @bitstreams//:manifest
 
 # The directory containing the bitstream is named after the git hash.
-bitstream_file=$(ci/scripts/target-location.sh ${bitstream_target})
-bitstream_commit=$(basename "$(dirname ${bitstream_file})")
+manifest_file=$(ci/scripts/target-location.sh @bitstreams//:manifest)
+
+bitstream_commit=$(./util/py/scripts/get_bitstream_build_id.py \
```

Comment:
Hm... might want to consider catching exceptions here. If the design doesn't exist in the cache for some reason, we should proceed to simply building it.

Reply:
Done. Now an empty `bitstream_commit` signifies that the design is not in the cache. Other errors will still trigger build failure.